python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -125,6 +138,13 @@ def runApPipe(metricsJob, workspace, parsedCmdLine):
     log = lsst.log.Log.getLogger('ap.verify.pipeline_driver.runApPipe')
 
     dataId = _parseDataId(parsedCmdLine.dataId)
+    #  After processes are implemented, remove the flake exception
+    processes = parsedCmdLine.processes  # noqa: F841
```

Comment:
I don't understand why you need these two lines at all.

Reply:
I didn't add them.  It was in there as a place holder (I assume), but because `processes` was not used, it failed linting.  I added the noqa so that flake8 would ignore that line until the `processes` functionality is implemented.

Reply:
Aha, it was removed in 0c57f674f4ccd5e22157ad0aaf67679b31ed7203, it must have gotten re-added during rebasing.